Casks/b/babeledit.rb

```diff
@@ -8,8 +8,8 @@
   homepage "https://www.codeandweb.com/babeledit"
 
   livecheck do
-    url "https://www.codeandweb.com/babeledit/download"
-    regex(%r{babeledit/download/v?(\d+(?:\.\d+)+)/mac-64}i)
+    url "https://www.codeandweb.com/releases/babeledit/updatecheck.php?b=no&o=mac&v=0.0.0"
```

Comment:
```suggestion
    url "https://www.codeandweb.com/releases/babeledit/updatecheck.php?b=no&o=mac"
```

I may be overlooking something but the response is identical if the `v` query string parameter is omitted or we use `v=0.0.0` (or `v=#{version}`). It seems like we can omit the parameter until/unless it makes a functional difference.